### ISSUE TITLE
VxMarkScan: Explore on-screen volume buttons

### ISSUE THREAD

**Issue by arsalansufi** [votingworks/vxsuite#5159]
https://votingworks.slack.com/archives/CJU9MSC6S/p1722278453909469
